patch/driver-support-optoe-write-max.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/drivers/misc/eeprom/optoe.c b/drivers/misc/eeprom/optoe.c
```

Comment:
Please add a `git format-patch` formatted patch with a commit message and Signed-off-by line.

Reply:
Added